packages/cli/src/lib/yargs-cli.ts

```diff
@@ -90,3 +94,15 @@ export function yargsCli<T = unknown>(
   // return CLI object
   return cli as unknown as Argv<T>;
 }
+
+function validatePersistFormat(persist: PersistConfig) {
+  try {
+    return (
+      persist?.format == null ||
+      persist.format.map(format => formatSchema.parse(format)).length ===
+        persist.format.length
+    );
+  } catch {
+    throw new Error('Unknown "persist.format" option. Use "md" or "json".');
```

Comment:
Would be more future-proof to print the allowed options from the same array that our validation uses (single source of truth), instead of hardcoding `"md" or "json"`.

Reply:
I agree. However, I got no clue how to get that option from the zod error :D any ideas?

Reply:
Update: Got it! It was unnecessarily tedious but I got the message 🎉